test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -57,9 +58,12 @@
     "test_AllenaiLongformerBase_repro_dynamic_shapes": TestFailure(
         ("cpu", "cuda", "xpu")
     ),
-    "test_conv_inference_heuristics_dynamic_shapes": TestFailure(("cuda", "xpu")),
     "test_randint_distribution_dynamic_shapes": TestFailure(("cuda", "xpu")),
 }
+if not torch._inductor.config.cpp_wrapper:
+    test_failures["test_conv_inference_heuristics_dynamic_shapes"] = TestFailure(
```

Comment:
Hmm, so this actually works in cpp wrapper but not python wrapper? Do we know why?

Reply:
I did not do a ton of digging, although I can if that's a condition for merge?

Reply:
@desertfire I'll investigate locally, and if anything of note results I'll put it in a later PR.